sdk/tables/azure-data-tables/inc/azure/data/tables/table_client.hpp

```diff
@@ -118,41 +103,6 @@ namespace Azure { namespace Data { namespace Tables {
         const std::string& tableName,
         const TableClientOptions& options = {});
 
-    /**
-     * @brief Initializes a new instance of tableClient.
-     *
-     * @param connectionString the connection string used to initialize.
-     * @param tableName The name of the table.
-     * @param options Optional client options that define the transport pipeline policies for
-     * authentication, retries, etc., that are applied to every request.
-     * @return TableClient.
-     */
-    static TableClient CreateFromConnectionString(
-        const std::string& connectionString,
-        const std::string& tableName,
-        const TableClientOptions& options = {});
-
-    /**
-     * @brief Retrieves details about any stored access policies specified on the table that may be
-     * used with Shared Access Signatures.
-     *
-     * @param context for canceling long running operations.
-     * @return Get access policy result.
-     */
-    Response<Models::TableAccessPolicy> GetAccessPolicy(Core::Context const& context = {});
```

Comment:
Why are we removing Get/Set AccessPolicy?

Reply:
Because credentials, they do not work with either token or SAS, connection string only, which we removed in this PR also. If we want these APIs to be there then we need those creds, which are not allowed anymore by SFI. so chicken <-> egg